Source/WebCore/Modules/webxr/WebXRSession.h

```diff
@@ -90,6 +90,9 @@ class WebXRSession final : public RefCounted<WebXRSession>, public EventTarget,
     bool supportsViewportScaling() const; 
     bool isPositionEmulated() const;
 
+    void applicationDidEnterBackground() { m_shouldServiceRequestVideoFrameCallbacks = true; }
```

Comment:
It's hard for me to grok why you have to enable RVAFC in the background, so this justifies a comment. Is it because Page's `updateRendering()` isn't running, so the immersive session needs to take over the responsibility to run RVAFC. Say that going into an immersive session puts the application in the background (because this is not obvious).

Reply:
Good idea! Added a comment.